.github/workflows/build.yml

```diff
@@ -17,6 +17,8 @@ jobs:
         run: npm install
       - name: Build
         run: npm run build
+      - name: Install dependencies
+        run: npm lint
```

Comment:
Wrong name, but it doesn't matter as I already added the check on main. Just update your branch.

Reply:
done